install/0000_00_cluster-version-operator_03_deployment.yaml

```diff
@@ -33,6 +33,7 @@ spec:
           - "--listen=0.0.0.0:9099"
           - "--serving-cert-file=/etc/tls/serving-cert/tls.crt"
           - "--serving-key-file=/etc/tls/serving-cert/tls.key"
+          - "--service-ca-bundle-file=/etc/tls/service-ca/service-ca.crt"
```

Comment:
Do we need this given it is the default?

Reply:
+1 to not explicitly setting this value to match the internal default.  One less place to change if the default ever needs to change.

Reply:
Thanks, it's removed in the new commit :+1: